src/openms/include/OpenMS/CONCEPT/Constants.h

```diff
@@ -279,6 +279,10 @@ namespace OpenMS
       */
       extern OPENMS_DLLAPI const std::string   FRAGMENT_ERROR_MEDIAN_PPM_USERPARAM;
 
+      extern OPENMS_DLLAPI const std::string   FRAGMENT_ERROR_PPM_METAVALUE_USERPARAM;
+
+      extern OPENMS_DLLAPI const std::string   FRAGMENT_ERROR_DA_METAVALUE_USERPARAM;
```

Comment:
- is this DA or m/z ?
- please also rename to remove redundant metavalue + userparam